src/client/cli/cmd/shell.cpp

```diff
@@ -117,12 +117,12 @@ std::vector<std::string> cmd::Shell::aliases() const
 
 QString cmd::Shell::short_help() const
 {
-    return QStringLiteral("Open a shell on a running instance");
+    return QStringLiteral("Open a shell on the primary or specified instance");
 }
 
 QString cmd::Shell::description() const
 {
-    return QStringLiteral("Open a shell prompt on the instance.");
+    return QStringLiteral("Open a shell prompt on the instance. If the instance is not running, it will automatically be started.");
```

Comment:
Good idea to mention this! Just a minor rephrasing suggestion:

```suggestion
    return QStringLiteral("Open a shell prompt on the instance. If the instance is not running, it will be started automatically.");
```